tests/core/CoreTest.py

```diff
@@ -383,7 +383,8 @@ def execute_run(bears):
         def on_result(result):
             results.append(result)
 
-        run(bears, on_result)
+        session = Session(bears, on_result)
+        session.run()
```

Comment:
what do you think about providing a shortcut? So we place a `run` function into the core module, that actually does this:
```
def run(bears, on_result):
    session = Session(bears, on_result)
    session.run()
```
For users this means one statement less (avoiding the `Session` instantiation) and we can maybe add more functions later (if that makes sense of course).